.github/workflows/main.yml

```diff
@@ -4,15 +4,18 @@ on:
   - pull_request
 jobs:
   test:
-    name: Node.js ${{ matrix.node-version }}
-    runs-on: ubuntu-latest
+    name: Node.js ${{ matrix.node-version }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}-latest
     strategy:
       fail-fast: false
       matrix:
         node-version:
           - 22
-          - 20
```

Comment:
Intentional?

Reply:
Yes. To reduce the number of CI tests since it is unlikely for a bug to be present in Node 20 but neither in Node 18 nor Node 22.

We can leave it though if you want?

Reply:
Nah, it's fine. Was just curious.